app.js

```diff
@@ -29,36 +29,45 @@ function checkResponseNo(response) {
     console.log('User response: ' + response + '. Wrong!');
   }
 }
-
-userResponse = prompt('Does Kyle like rain drops on roses?');
-console.log('Does Kyle like rain drops on roses?');
-checkResponseYes(userResponse);
-numQuestions++;
-
+function questionOne(){
+  userResponse = prompt('Does Kyle like rain drops on roses?');
+  console.log('Does Kyle like rain drops on roses?');
+  checkResponseYes(userResponse);
+  numQuestions++;
+}
+questionOne();
+function questionTwo(){
 userResponse = prompt('Does Kyle like whiskers on kittens?');
 console.log('Does Kyle like whiskers on kittens?');
 checkResponseYes(userResponse);
 numQuestions++;
-
+}
+questionTwo();
+function questionThree(){
```

Comment:
You should keep your indentation consistent. Best-practice is to indent anything that is contained in something else one tab from the parent/container.